### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by till**
Small updates on how to run Clouseau in a production environment.

h/t @rnewson :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,68 @@
...
+```
 mvn scala:run -Dlauncher=clouseau3
+```
+
+## Running in production
+
+Build Clouseau and move to `/opt/clouseau`:
+
+```
+cd clouseau-clone
```
Comment (jiangphcn) [inline README.md]: after git repo is cloned, the default directory name is `clouseau` instead of `clouseau-clone`.

Comment (till) [reply README.md]: @jiangphcn Done!

**Comment by jiangphcn** [comment]
+1 from my side. Please squash before merge. Thanks

**Comment by till** [comment]
@jiangphcn Done! Thanks for your feedback!

**Comment by till** [comment]
Guess this didn't happen. 😂

**Pull request closed without merging**